src/styles/stopwatch.scss

```diff
@@ -0,0 +1,67 @@
+.stopwatch {
+  position: relative;
+  width: $stopwatch-width;
+  height: $stopwatch-height;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: 50%;
+    transform: translateX(-50%);
+    transform-origin: bottom center;
+    animation-name: rotate;
+    animation-duration: 3600s;
+    animation-timing-function: steps(60);
+    animation-iteration-count: infinite;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: $seconds-hand-height;
+    width: $hand-width-seconds;
+    background-color: $seconds-hand-color;
+    bottom: 50%;
+    left: 50%;
+    transform: translateX(-50%);
+    transform-origin: bottom center;
```

Comment:
Similar to the `.stopwatch__minutes-hand`, the `.stopwatch__seconds-hand` also has the `transform` property declared twice. Combine the translate and transform-origin into a single transform property to avoid one declaration overriding the other.